cogs/commands/chatgpt.py

```diff
@@ -80,6 +89,11 @@ async def on_message(self, message: discord.Message):
         await self.cmd(ctx)
 
     async def cmd(self, ctx: Context):
+        if is_user_banned_openai(ctx.author.id):  # if user is banned error
+            return await ctx.message.reply(
+                "You are banned from using openAI commands, please contact an exec if you think this is a mistake"
```

Comment:
This message is repeated quite a few times, maybe refactor into an `openai_ban_error` or something